fhir-codegen/gradle-plugin/src/main/kotlin/com/google/fhir/codegen/schema/FhirEnum.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.fhir.codegen.schema
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class FhirEnum(
+  val description: String? = null,
+  val constants: List<FhirEnumConstant> = emptyList(),
+) {
+  fun getSystem(): String? {
+    return constants.firstOrNull()?.system
```

Comment:
is this correct? does an enum always have the only one code system?

seems to me it can include multiple code systems.